cmd/kubeadm/app/cmd/upgrade/node.go

```diff
@@ -105,11 +106,26 @@ func newCmdNode(out io.Writer) *cobra.Command {
 
 // newNodeOptions returns a struct ready for being used for creating cmd kubeadm upgrade node flags.
 func newNodeOptions() *nodeOptions {
+	kubeConfigPath := constants.GetKubeletKubeConfigPath()
+
+	// isControlPlane checks if a node is a control-plane node by looking up
```

Comment:
The comment refers to `isControlPlane`, while the following term is `isControlPlaneNode`. Although both terms are acceptable, I prefer using `isControlPlane` as it is shorter.  Not strongly required, but for consistency, please keep the same one.